testing/wiremock-shaded-for-testing/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+plugins {
+  id("com.gradleup.shadow")
+  id("otel.java-conventions")
+}
+
+dependencies {
+  implementation("com.github.tomakehurst:wiremock-jre8:2.35.2")
+  implementation("com.google.errorprone:error_prone_annotations")
+  implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.18.2")
+}
+
+tasks {
+  shadowJar {
+    dependencies {
+      exclude(dependency("org.slf4j:slf4j-api"))
+      exclude(dependency("org.junit.jupiter:junit-jupiter-api"))
+      exclude(dependency("org.junit.platform:junit-platform-commons"))
+      exclude(dependency("org.ow2.asm:asm"))
+      // Exclude dependencies bundled during Armeria shading
+      exclude(dependency("com.fasterxml.jackson.core:jackson-annotations"))
+      exclude(dependency("com.fasterxml.jackson.core:jackson-core"))
+      exclude(dependency("com.fasterxml.jackson.core:jackson-databind"))
+    }
+
+    // Ensures tests are not affected by wiremock dependencies. Wiremock itself is
+    // safe with its original name.
+    relocate("com.google.common", "io.opentelemetry.testing.internal.guava")
```

Comment:
It was tedious to find all these, but I couldn't find a better way. I thought it could be nice to just relocate everything, but I couldn't see any opt-out in that case to not relocate slf4j / junit. So opted-in with a big list instead.

Reply:
Any notes to add on regenerate/check assumptions on this when libs update?